scripts/assessment-cof-reporting_scores.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+	application_id AS "Application ID",
+	sub_criteria_id AS "Score Subcriteria",
+	score AS "Score",
```

Comment:
This is incorrect I believe as we only want the latest score per subcriteria else we risk including scores which have since been rescored.